int32 new_id
---
bool success